data/json/professions.json

```diff
@@ -6265,5 +6305,605 @@
+  },
+  {
+    "type": "profession",
+    "id": "sawbones",
+    "name": "Sawbones",
+    "description": "There was never a shortage of tough guys in need of patching up or organs in need of removal.  You may have never had a medical license, but that doesn't matter to anyone now that the world has ended.",
+    "points": 3,
+    "skills": [
+      { "level": 5, "name": "firstaid" },
+      { "level": 4, "name": "speech" },
+      { "level": 2, "name": "melee" },
+      { "level": 2, "name": "cutting" }
+    ],
+    "proficiencies": [ "prof_wound_care", "prof_wound_care_expert", "prof_intro_biology", "prof_physiology", "prof_dissect_humans" ],
+    "items": {
+      "both": {
+        "items": [ "dress_shirt", "gloves_medical", "socks", "dress_shoes", "wristwatch", "syringe", "morphine" ],
+        "entries": [
+          { "item": "pants", "variant": "pants_black" },
+          { "item": "mask_dust", "variant": "white_mask_dust" },
+          { "group": "charged_smart_phone" },
+          { "item": "scalpel", "custom-flags": [ "auto_wield" ] }
+        ]
+      },
+      "male": [ "boxer_shorts" ],
+      "female": [ "bra", "panties" ]
+    }
+  },
+  {
+    "type": "profession",
+    "id": "chauffeur",
+    "name": { "male": "Chauffeur", "female": "Chauffeuse" },
+    "description": "You were the personal driver of an A-list celebrity.  When shit hit the fan, you decided to speed off instead of waiting for them.  Now it's just you and your limo.",
+    "points": 3,
+    "skills": [ { "level": 6, "name": "driving" }, { "level": 3, "name": "speech" } ],
+    "vehicle": "limousine",
+    "items": {
+      "both": {
+        "items": [ "suit", "gloves_light", "socks", "dress_shoes", "wristwatch", "tie_bow" ],
+        "entries": [ { "group": "charged_smart_phone" } ]
+      },
+      "male": [ "boxer_shorts" ],
+      "female": [ "bra", "panties" ]
+    }
+  },
+  {
+    "type": "profession",
+    "id": "bouncer",
+    "name": "Bouncer",
+    "description": "Your years of working as a nightclub bouncer have taught you a thing or two about dealing with rowdy clubbers.  It's going to take more than a stern staredown to handle a zombie though.",
+    "points": 3,
+    "skills": [
+      { "level": 4, "name": "melee" },
+      { "level": 4, "name": "unarmed" },
+      { "level": 3, "name": "dodge" },
+      { "level": 3, "name": "speech" }
+    ],
+    "traits": [ "TERRIFYING" ],
+    "items": {
+      "both": {
+        "items": [ "socks", "sneakers", "wristwatch", "fancy_sunglasses" ],
+        "entries": [
+          { "group": "charged_smart_phone" },
+          { "item": "pants", "variant": "pants_black" },
+          { "item": "tshirt", "variant": "generic_tshirt" }
+        ]
+      },
+      "male": [ "boxer_shorts" ],
+      "female": [ "bra", "panties" ]
+    }
+  },
+  {
+    "type": "profession",
+    "id": "idol",
+    "name": "Pop Star",
```

Comment:
I wouldn't add something like this unless you first want to go through and add some npc reactions to you being famous. Which would be pretty awesome and fun, but should be a PR of its own.